### DISCUSSION TITLE
[RFC] Install from/filter to only install required packages/dependencies

### DISCUSSION THREAD

**Discussion by bkniffler** [vercel/turborepo#7495] [Ideas]
### Describe the feature you'd like to request

Being able to build with dependency graph is cool. But what if, in a big repo with many different projects, I'd only like to install & build the frontend project without having the backend dependencies. Rush is able to do just that. Is there a way to accomplish the same with turbo repo?

### Describe the solution you'd like

`turbo install --filter=my-app`

### Describe alternatives you've considered

Using rush; or using pnpm and deleting the folders that I don't want to have included, just before installing.

**Comment by qivox** [comment]
I would also be looking forward to something like this (#1334)

**Comment by qivox** [comment]
Any updates on this one?

**Comment by mehulkar** [comment]
we've thought about this a few times over the years, will take into consideration! In the meantime, [`turbo prune`](https://turbo.build/repo/docs/reference/command-line-reference/prune) may help.
